packages/contracts/tests/MACI.test.ts

```diff
@@ -25,7 +25,7 @@ import {
 import { deployTestContracts } from "./utils";
 
 describe("MACI", function test() {
-  this.timeout(90000);
+  this.timeout(900000); // 15 minutes
```

Comment:
these are the only changes required to work with zksync? why? 😋

Reply:
No, there are other changes in this PR. Increasing timeout is needed to run tests on forks because they are slower than local hardhat node.